packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -4,6 +4,20 @@ title: Okta API Products Release Notes 2021
 
 ## May
 
+### Weekly Release 2021.05.2
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2021.05.2](#bugs-fixed-in-2021-05-2)         | May 19, 2021           |
+
+#### Bugs fixed in 2021.05.2
+
+* When a user was enrolled with SMS or Call factors and an additional or retry [API call](/docs/reference/api/factors) was made within 30 seconds of enrollment, the response included multiple rate limit headers. (OKTA-379654)
+
+* When [OIDC apps](/docs/reference/api/oauth-clients/#register-new-client) were created concurrently, some apps were created in a deactivated state. (OKTA-384407)
+
+* The [Client Credentials Flow](/docs/guides/customize-authz-server/overview/) could not implement a custom claim named `scope`. (OKTA-389874)
```

Comment:
Please check the Client Credentials Flow link?
```suggestion
* The [Client Credentials Flow](/docs/guides/implement-client-creds/overview/) could not implement a custom claim named `scope`. (OKTA-389874)
```

Reply:
Thanks, Van. Ultimately, you are redirected to the Create an Authorization Server Guide / Create Scopes section for instructions; however, could be a little confusing. Will commit your suggestion. Thanks!